ckcp/openshift_dev_setup.sh

```diff
@@ -13,16 +12,14 @@ GITOPS_DIR="$(dirname "$SCRIPT_DIR")/gitops"
 CKCP_DIR="$(dirname "$SCRIPT_DIR")/ckcp"
 WORK_DIR="$SCRIPT_DIR/work"
 KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
-KUBECONFIG_KCP="$WORK_DIR/kubeconfig/admin.kubeconfig"
-KUBECONFIG_MERGED="merged-config.kubeconfig:$KUBECONFIG:$KUBECONFIG_KCP"
+KUBECONFIG_KCP="$WORK_DIR/credentials/kubeconfig/kcp/admin.kubeconfig.base"
 CR_TOSYNC=(
```

Comment:
I know you didn't work on this, but could you please change this to CR_TO_SYNC?

Reply:
Done